aptos-move/framework/aptos-framework/sources/aptos_account.spec.move

```diff
@@ -217,4 +226,17 @@ spec aptos_framework::aptos_account {
         aborts_if type_info::type_of<CoinType>() != type_info::type_of<AptosCoin>()
             && !coin::is_account_registered<CoinType>(to) && !can_receive_direct_coin_transfers(to);
     }
+
+    spec schema TransferEnsures<CoinType> {
+        coin_store_to: coin::CoinStore<CoinType>;
+        coin_store_source: coin::CoinStore<CoinType>;
+        amount: u64;
+
+        let o_balance_source = coin_store_source.coin.value;
```

Comment:
nit: could you explain the name "o_" and "p_"?

Reply:
Oh, it means the origin and the post, should I add comment?

Reply:
maybe rename to "origin_..." and "post_" to make it more clear?

Reply:
Fixed